From c79bc54a05f36ab5ecd13a3998b955b71ab47eb0 Mon Sep 17 00:00:00 2001
From: dhacker29 <davidhackerdvm@gmail.com>
Date: Thu, 22 Aug 2013 22:28:06 -0500
Subject: [PATCH] Vold: Allow Settings to determine if a switchable pair exists
 (1/2)

This will eliminate the need for pre-configured device pairs listed
in the build.prop, and display the setting if a pair exists.
This exapnds on commit dc8d157d73fb88ee33b38e724a8aabe7d5e53eee

Change-Id: I5f15a711b5ceabddaf842c252d890545f40b4786
---
 Volume.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Volume.cpp b/Volume.cpp
index c175f93..3db6c43 100644
--- a/Volume.cpp
+++ b/Volume.cpp
@@ -152,7 +152,7 @@ Volume::Volume(VolumeManager *vm, const char *label, const char *mount_point) {
         char *first, *second = NULL;
         const char *delim = ",";
 
-        property_get("ro.vold.switchablepair", switchable, "");
+        property_get("persist.sys.vold.switchablepair", switchable, "");
 
         if (!(first = strtok(switchable, delim))) {
             SLOGE("Mount switch requested, but no switchable mountpoints found");
-- 
1.8.3.msysgit.0

